tox.ini

```diff
@@ -41,14 +41,17 @@ commands =
     black {posargs:.}
 
 [testenv:test]
+setenv =
+    PIP_FIND_LINKS = https://girder.github.io/large_image_wheels
 passenv =
     DJANGO_CELERY_BROKER_URL
     DJANGO_DATABASE_URL
     DJANGO_MINIO_STORAGE_ACCESS_KEY
     DJANGO_MINIO_STORAGE_ENDPOINT
     DJANGO_MINIO_STORAGE_SECRET_KEY
+extras =
+    dev
 deps =
-    -rrequirements.txt
     factory-boy
     faker<=5.4.0
```

Comment:
I'm also curious, why was `faker` pinned?

Reply:
I ran into a bug after one of their releases. See https://github.com/ResonantGeoData/ResonantGeoData/pull/236#discussion_r555914603